File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/UforeEkstraUT.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.beregning.BeregningUforeperiode;
import no.nav.domain.pensjon.regler.grunnlag.Uforeperiode;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/* Ekstra informasjon til beregnet uføretrygd. Brukes for at PREG skal beregne en uførehistorikk for uføretrygd.
 * 
 * @author Swiddy de Louw (Capgemini) - PK-10228
 */
public class UforeEkstraUT implements Serializable {
    /*
     * Beregnede uføreperioder for uføretrygd.
     * Dette er uføreperioder som beregnes av PREG, og ligger ved beregningen for en uføretrygd.
     */
    private List<BeregningUforeperiode> beregnetUforeperiodeListe;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/UforeEkstraUT.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.beregning.BeregningUforeperiode
import java.io.Serializable

class UforeEkstraUT : Serializable {
    /**
     * Beregnede uføreperioder for uføretrygd.
     * Dette er uføreperioder som beregnes av pensjon-regler, og ligger ved beregningen for en uføretrygd.
     */
    var beregnetUforeperiodeListe: List<BeregningUforeperiode> = mutableListOf()
}